test/test_evaluation/test_train_evaluator.py

```diff
@@ -612,8 +613,11 @@ def test_subsample_indices_regression(self, mock, backend_mock):
 
         # Corner cases
         evaluator.subsample = 0
-        train_indices5 = evaluator.subsample_indices(train_indices)
-        np.testing.assert_allclose(train_indices5, np.array([]))
```

Comment:
This behavior has changed in sklearn 0.21. Now a train_size=0 will raise an exception. Does autosklearn make any assumptions for this case? In other words, do we have to edit TrainEvaluator.subsample_indices() to return np.array([]) when self.subsample=0?